src/api/app/controllers/person/token_controller.rb

```diff
@@ -50,5 +51,18 @@ def set_package
 
       @package = Package.get_by_project_and_name(params[:project], params[:package])
     end
+
+    def validate_operation
+      operation_param = params[:operation]
+      # TODO: align operation parameter allowed values
+      # - webUI: https://github.com/openSUSE/open-build-service/blob/master/src/api/app/models/token.rb#L27
+      # - API: https://github.com/openSUSE/open-build-service/blob/master/src/api/public/apidocs/paths/person_login_token.yaml#L89
+      return if operation_param.nil? ||
+                ['runservice', 'rebuild', 'release'].include?(operation_param) # possible API parameter values
```

Comment:
```suggestion
                ['runservice', 'rebuild', 'release', 'workflow'].include?(operation_param) # possible API parameter values
```
We also support the `workflow` value for the `operation` parameter, as the failing test pointed out. It is not yet documented, but we support it.